### DISCUSSION TITLE
Release v1.2.4

### DISCUSSION THREAD

**Discussion by daledavies** [daledavies/jump#44] [Announcements]
### Added
- Issue #41: Add support for a custom favicon.
- Add new default favicon.

### Fixed
- Refactor API implementation.
- Various other code improvements.

<hr /><em>This discussion was created from the release <a href='https://github.com/daledavies/jump/releases/tag/v1.2.4'>Release v1.2.4</a>.</em>

**Comment by Toucani** [comment]
Hey, just wanted to say thank you for the favicon feature! I wanted it from the beginning but wasn't bold enough to ask for it:)

**Reply by daledavies** [reply]
Hey no problem, thanks for getting in touch now :)
